config/redirects.txt

```diff
@@ -127,6 +121,15 @@
 /learn/more/examples/seed-projects /learn/more/examples/labs-projects
 
 
+# Important: keep dynamic redirect below the static redirects
+# https://developers.cloudflare.com/pages/platform/redirects/
+# Dynamic redirects
+
+/deployment/network/:version/ /reference/OS/network/
+/understanding/understanding-devices/:version/ /reference/OS/overview/
+/reference/OS/overview/:version/ /reference/OS/overview/
+/reference/OS/network/:version/ /reference/OS/network/
```

Comment:
These are new, I'll need to update my regex to ignore these.

Reply:
Have always been there?

Reply:
Oh, I never noticed the colons before. Do they also need both slash and non-slash variants?

Reply:
![Screenshot from 2023-01-11 01-18-16](https://user-images.githubusercontent.com/22801822/211648231-83ddf00a-96e2-4e62-aee0-b5020e976124.png)

We can improve the regex but works just fine ^^